en/app.ftl

```diff
@@ -0,0 +1,270 @@
+banner-download-install-extension-cta = Add { -brand-name-relay } to { -brand-name-firefox }
+banner-upgrade-headline = Upgrade to { -brand-name-relay-premium }
+banner-upgrade-copy = { -brand-name-firefox } { -brand-name-relay-premium } makes creating email aliases even easier, with custom alias domains and unlimited aliases.
+banner-upgrade-cta = Upgrade to { -brand-name-relay-premium }
+banner-choose-subdomain-headline = Choose your own domain
+banner-choose-subdomain-headline-aliases = Get your own custom alias domain
+banner-choose-subdomain-copy = You can choose a custom domain for your email aliases.
+banner-choose-subdomain-warning = Note: You cannot change your domain later
+banner-choose-subdomain-input-placeholder = Search domain
+banner-choose-subdomain-submit = Get Domain
+banner-pack-upgrade-headline-html = Upgrade to <strong>{ -brand-name-firefox } { -brand-name-relay-premium }</strong> to get more aliases
+banner-pack-upgrade-copy = With unlimited email aliases and your own email domain, { -brand-name-firefox } { -brand-name-relay-premium } helps you stay protected online.
+banner-pack-upgrade-cta = Upgrade Now
+
+# This string is followed by name (string) that the user chooses
+banner-choose-subdomain-label = Your domain is:
+
+# Variables:
+# $subdomain (url) - User-set subdomain
+banner-choose-subdomain-label = You can make up any address @{ $subdomain }
+
+## Error Messages
+
+# Variables:
+#   $number (string) - Maximum number of aliases a user can make on a free account. 
+error-premium-set-make-aliases = You must be a premium subscriber to make more than { $number } aliases
+error-premium-cannot-change-subdomain = You cannot change your subdomain
+error-premium-set-subdomain = You must be a premium subscriber to set a subdomain
+error-premium-set-create-subdomain = You must be a premium subscriber to create subdomain aliases
+error-subdomain-not-created = Subdomain could not be created, try something else
+error-subdomain-email-not-created = Email address with subdomain could not be created, try something else
+error-subdomain-select = You must select a subdomain before creating a subdomain aliases
+
+## Onboarding 
+
+onboarding-headline = Create your first alias, you have three ways…
+onboarding-tip-1 = Just click the button “Generate New Alias” to create your first alias
+onboarding-tip-2 = By selecting the { -brand-name-firefox-relay } icon when it appears in email fields
+onboarding-tip-3 = Via the context menu, just right-click (Windows) or Control-click (macOS) on form fields to access the menu and generate an alias
+
+## Alias Modals
+
+modal-rename-alias-saved = Label saved!
+modal-delete-headline = Permanently delete this alias?
+modal-delete-warning-recovery-html = Once you delete this alias, it cannot be recovered. 
+    { -brand-name-firefox-relay } will no longer forward messages sent to <strong></strong>, including messages that allow you to reset lost passwords.
```

Comment:
what  is between `<strong></strong>`

Reply:
The email address that will be deleted. It's added via JavaScript (rather than passed as a variable when calling the translated string). Would you like me to add a note? 

<img width="560" alt="image" src="https://user-images.githubusercontent.com/2692333/129618291-281ace28-94f3-47d7-b051-7ae909184358.png">

Reply:
Alternately, I could set it up as a variable so that localizers would understand the context and just not use it (in that specific way). That may be the best course of action.

Reply:
I went ahead and did exactly that. See revision.